pdfminer/layout.py

```diff
@@ -50,7 +50,9 @@ class LAParams:
     :param boxes_flow: Specifies how much a horizontal and vertical position
         of a text matters when determining the order of text boxes. The value
         should be within the range of -1.0 (only horizontal position
-        matters) to +1.0 (only vertical position matters).
+        matters) to +1.0 (only vertical position matters). You can also pass
+        `None` to disable advanced layout analysis, and instead return text
+        based on the (x, y) coordinates of the bottom left corner.
```

Comment:
> based on the (x, y) coordinates of the bottom left corner

Took me 3 times to understand that it is the "bottom left corner" _of the text boxes_.

Reply:
I'll re-word it :)